src/jira/client/index.ts

```diff
@@ -280,11 +281,45 @@ const batchedBulkUpdate = async (
 	logger?: Logger,
 	options?: { preventTransitions: boolean }
 ) => {
-	// Initialize with an empty chunk of commits so we still process the request if there are no commits in the payload
-	const commitChunks: JiraCommit[][] = _.chunk(dedupCommits(data.commits), 400);
+	if (booleanFlag(BooleanFlags.REMOVE_WHILE_LOOP, true)) {
```

Comment:
Should the default value be false? Otherwise, this code would roll out if LaunchDarkly has an outage

Reply:
but then it won't be enabled by default on your local...